SpecializedServers/MultiSocks/DirtySocks/Messages/QuikIn.cs

```diff
@@ -5,11 +5,14 @@ public class QuikIn : AbstractMessage
         public override string _Name { get => "quik"; }
         public string? KIND { get; set; }
         public string? QMFP { get; set; }
-
+        public string? AUTO { get; set; }
+        public string? VOIP { get; set; }
+        public string? GAME { get; set; }
+        public string? CHAR { get; set; }
         public override void Process(AbstractDirtySockServer context, DirtySockClient client)
         {
-            client.SendMessage(new QuikOut());
```

Comment:
This will break Burnout Paradise, need a specific check here for older dirtysocks clients.

Reply:
Ahhh i see, well acoording to the docs, if no game is found, it should be possible to return a game create